particle-iot/device-os
# [wiring] Enable Wiring_SetupButtonUX feature for Boron

### Problem

Enable Wiring_SetupButtonUX feature for Boron

### Solution

N/A

### Steps to Test

1. Single-click mode button to check RSSI by counting how many times the RGB LED blinks
2. Double-click mode button to power off the device 

### Example App
N/A

### References

[CH37322]

---

### Completeness
